### DISCUSSION TITLE
Pixi developer at home renovation startup (full-time or part-time)

### DISCUSSION THREAD

**Discussion by c-w** [pixijs/pixijs#10093] [Jobs]
[Homebase Automation](https://www.linkedin.com/company/builtwithnook/) is searching for a full-time or part-time Pixi developer to help us build tools for home owners.

Approximately $450b is spent annually on home improvement in the U.S. We’re tapping into this market by building tools that empower homeowners to manage the complexity of the renovation lifecycle. This market is vastly underserved, with existing offerings targeting industry professionals as opposed to homeowners. Our tools put everyday people back in charge of what is happening under their roof.

Concretely, you'll be working on an interactive floor plan tool that enables home-owners to onboard their rooms and spaces onto our platform, plan their renovations, and communicate their project ideas with trades. Our codebase uses Typescript, React, XState, and Pixi.

You can work remote from anywhere as long as you have a few hours of overlap with business hours in GMT-5 for the occasional call with our design and product team.

Our interview process aims to respect your time: async technical discussion with our CTO about past experiences and engineering philosophy, paid take-home coding exercise representative of our work, final conversation with CEO and offer.

Interested? Reach out to jobs@homebase.llc

**Comment by dean345677** [comment]
This sounds like a great opportunity for Pixi developers looking to create tools that simplify home renovation projects. Managing a renovation can be complex, and having platforms that allow homeowners to visualize their ideas, plan their spaces, and communicate effectively with contractors is extremely valuable. PHS Renovations, for example, is a top renovation company in Winnipeg that has been helping homeowners bring their visions to life since 2021. They specialize in professional [home renovation](https://phsrenovations.ca/) services, from custom kitchens and bathrooms to full-home remodels, ensuring every project is executed with care and attention to detail. Their team focuses on making the process stress-free, guiding clients from initial planning to the final result so that every space turns out exactly as imagined. Combining such expert services with innovative tools like the one Homebase Automation is developing shows how technology and professional experience together can make home renovations easier and more enjoyable for homeowners.
